public/static-checkout/css/style.css

```diff
@@ -0,0 +1,256 @@
+}
+
+.user-button {
+  top: 50%;
+  position: absolute;
+  left: 90%;
+  transform: translate(-50%, -50%);
+  float: right;
+  cursor: pointer;
+  background-color: #eee;
+  border-color: #eee;
+  border-radius: 50rem;
+  border: 2px solid transparent;
+  padding: 0.375rem 1rem 0.375rem 1rem;
+  text-decoration: none;
+  display: inline-flex;
+  vertical-align: middle;
+}
+
+.user-button span {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  padding-left: 0.5rem;
+}
+
+.dropdown-menu {
+  display: none;
+  padding: .5rem 0;
+  box-shadow: 0 .125rem .25rem rgba(0,0,0,.3);
+  margin: 0.125rem 0 0;
+  border: 1px solid rgba(0,0,0,.15);
+  border-radius: 0.3rem;
+  font-size: 1rem;
+  background: #fff;
+  width: 158px;
+  float: right;
+  margin-top: 3.2rem;
+  margin-right: 1rem;
+}
+
+.show {
+  display: block;
+}
+
+.dropdown-item {
+  display: block;
+  text-decoration: none;
+  width: 100%;
+  white-space: nowrap;
+  padding: 0.25rem 1.5rem;
+  clear: both;
+  text-align: inherit;
+  background-color: transparent;
+  border: 0;
+}
+
+.dropdown-item:hover {
+  background-color: #f9f9f9;
+}
+
+.container-fluid {
+  margin: auto;
+  max-width: 1440px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.payment-container {
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+}
+
+.cart {
+  border-right: 1px solid #b9babe;
+  padding-left: 15px;
+  padding-right: 48px;
+}
+
+.checkout {
+  float: right !important;
+  padding-right: 15px;
+  padding-left: 48px;
+}
+
+.payment-method button {
+  margin-right: .5rem;
+  padding: 0.5rem;
+  margin-bottom: 0.5rem;
+  border-radius: 0.3rem;
+  height: 44px;
+  cursor: pointer;
+}
+
+.active-button {
+  border-bottom: .1875rem solid #23419f;
+}
+
+.payment-method img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.payment-container input {
+  padding: .375rem .75rem;
+  background-color: #fff;
+  background-clip: padding-box;
+  border: 1px solid #b9babe;
+  border-radius: 0.3rem;
+  display: block;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #2d323e;
+  width: 23rem;
+}
+
+.submit-button {
+  color: white;
+  line-height: 1.5;
+  display: block;
+  vertical-align: middle;
+  background-color: #23419f;
+  border-color: #23419f;
+  padding: 0.5rem 1.75rem;
+  border-radius: 50rem;
+  text-align: center;
+  border: 2px solid;
+}
+
+.img-thumbnail {
+  border: 1px solid #b9babe;
+  padding: 0.25rem;
+  background-color: #fff;
+  border-radius: 0.3rem;
+  width: 40%;
+}
+
+.product-line-item {
+  padding-left: 2rem;
+  vertical-align: middle;
+  top: 50%;
+  left: 50%;
+  transform: translate(0%, 25%);
+}
+
+.grid-container {
+  width: 100%;
+  max-width: 1380px;
+}
+
+.checkout-section {
+  margin-bottom: 3rem;
+}
+
+.col-right {
+  float: right;
+}
+
+.col-left {
+  float: left;
+}
+
+.row:before,
+.row:after {
+  content:"";
+  display: table ;
+  clear:both;
+}
+
+[class*='col-'] {
+  display: inline-block;
+  min-height: 1px;
+}
+
+.col-1 {width: 8.33%;}
```

Comment:
it looks like for now most of these classes are unused and could be removed? (unless they become necessary for making things responsive)